### DISCUSSION TITLE
同样ETL的insert语句，升级到2.1.2后似乎变慢非常多

### DISCUSSION THREAD

**Discussion by joeykoer** [apache/doris#33986] [Show and tell]
昨天尝试给你集群从2.1.1升级到2.1.2，早上发现凌晨的ETL作业执行未完成。关掉重跑后，仍然处于CPU高消耗且持续半小时没结束，平时一般只要1分钟就结束了。退回版本后，重跑果然一分钟结束。
![image](https://github.com/apache/doris/assets/163825215/1391cad1-0b3e-42b1-80dd-ce38fa0e8b5c)
这里是有什么配置项需要优化吗？

**Comment by yagagagaga** [comment]
方便把SQL发出来看看吗
